### ISSUE TITLE
Specified tax amount is invalid.

### ISSUE THREAD

**Issue by IbrahimS2** [Laybuy/Magento-1#6]
I simply integrated the extension without any problem, but upon clicking "place order" there is an error where it prevents the customers from reaching laybuy page

> Sorry there was an Error redirecting you to Laybuy: ERROR (Specified tax amount is invalid.)

When I reviewed the logs I have seen this:

```
2019-03-21T02:31:16+00:00 DEBUG (7): 5c92f7732f2c3 --  Laybuy_Payments_Model_Payments::setupLaybuy CLIENT INIT: 104052:UWgejQo7E42nZV4TMTzXPwngenZdNSQRr2w94VMGRS1V3hBsgm78sqJG8xesTcqg
2019-03-21T02:31:16+00:00 DEBUG (7): 5c92f7732f2c3 --  Laybuy_Payments_Model_Payments::setupLaybuy SETUP COMPLETE 
2019-03-21T02:31:17+00:00 DEBUG (7): 5c92f7732f2c3 --  stdClass Object
(
    [result] => ERROR
    [error] => Specified tax amount is invalid.
)

2019-03-21T02:31:17+00:00 DEBUG (7): 5c92f7732f2c3 --  LAYBUY: ORDER CREATE FAILED (ERROR -> Specified tax amount is invalid.) 
2019-03-21T02:31:17+00:00 DEBUG (7): 5c92f7732f2c3 --  LAYBUY: ORDER CREATE FAILED, found quote 213151

```

Now I am not quite sure if this a problem from our side or laybuy servers not taking ZERO Taxes value.

**Comment by IbrahimS2** [comment]
I have made additional debugging and from what I can tell that there is no problem with the main data, and it is sending all the proper data etc..., but the response from Laybuy coming back with Tax Error message as mentioned above!

**Comment by IbrahimS2** [comment]
This has been resolved based on the following commit 2d5c7123d02c3af9567089414e715f83768f94aa
